### DISCUSSION TITLE
arm64 linux runners

### DISCUSSION THREAD

**Discussion by JanneKiiskila** [community/community#68340] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

Hei,

would it be possible to get `arm64` (aka aarch64) architecture based GitHub hosted runners in ubuntu-22.04 and ubuntu-latest? Now there's only Intel/AMD architectures supported and that's quite not enough.

**Comment by Qard** [comment, marked as answer]
See: https://github.com/orgs/community/discussions/19197
